packages/fetchai/connections/p2p_libp2p/dht/dhtpeer/dhtpeer.go

```diff
@@ -831,7 +837,16 @@ func (dhtPeer *DHTPeer) registerAgentAddress(addr string) error {
 	linfo().Str("op", "register").
 		Str("addr", addr).
 		Msgf("Announcing address to the dht with cid key %s", addressCID.String())
+	//var elapsed time.Duration
+	//start := time.Now()
 	err = dhtPeer.dht.Provide(ctx, addressCID, true)
+	//elapsed = time.Since(start)
```

Comment:
Do we need this commented code here and above? If yes maybe it worth to have a comment that explains why it's here.

Reply:
addressed